src/model_io/urdf/include/private/URDFModelExport.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2017 Fondazione Istituto Italiano di Tecnologia
+ * Authors: Silvio Traversaro
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ */
+
+#ifndef IDYNTREE_URDF_MODEL_EXPORT_H
+#define IDYNTREE_URDF_MODEL_EXPORT_H
+
+#include <string>
+
+#include <iDynTree/ModelIO/ModelExporter.h>
+
+namespace iDynTree
+
+{
+
+class Model;
+
+/**
+ * \ingroup iDynTreeModelIO
+ *
+ * Export a iDynTree::Model object to a URDF file.
+ *
+ * @see ModelExporterOptions for more details on supported options.
+ *
+ * @warning This function does not support exporting sensor or solid shapes at the moment.
+ *
+ * @param[in] options the URDFParserOptions struct of options passed to the parser
```

Comment:
It'd be great to document also the output and default parameters here. Also, the location in which the urdf fille will be saved.

Reply:
For the default options, the user is referred to the iDynTree::ModelExportOptions, that contains docs related to the available and the default options. I will make it clear in the commit to address reviews, let me know if it is enough.